packages/inspector/src/handler.ts

```diff
@@ -60,14 +60,32 @@ export class InspectionHandler implements IDisposable, IInspector.IInspectable {
     }
 
     if (this._editor && !this._editor.isDisposed) {
-      this._editor.model.value.changed.disconnect(this.onTextChanged, this);
+      this._selectionsMonitor.activityStopped.disconnect(
+        this.onEditorChange,
+        this
+      );
+      this._valueMonitor.activityStopped.disconnect(this.onEditorChange, this);
     }
     let editor = (this._editor = newValue);
     if (editor) {
       // Clear the inspector in preparation for a new editor.
       this._cleared.emit(void 0);
-      editor.model.value.changed.connect(
-        this.onTextChanged,
+
+      this._selectionsMonitor = new ActivityMonitor({
+        signal: editor.model.selections.changed,
+        timeout: 1000
```

Comment:
My feeling is this is maybe a *bit* long to wait for the inspector to trigger. Maybe 250 or 500 ms?

Reply:
Yup, done in last commit.